README

```diff
@@ -15,74 +25,74 @@ Evaluation metric currently in xapian
+It's compulsory to set relfile and runfile for using this option.
 
 Configuring evaluation module
 ------------------------------
 
 Configurable Parameters:
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-	 - textfile     - path/filename of text file to be indexed for evaluation
-	 - language     - corpus language to be indexed for stemming 
-	 - db           - path of database which is index of above textfile
-	 - querytype    - type of query
-	 - queryfile    - path/filename of query file to be ran and searched
-	 - resultsfile  - path/filename of results file(run file)
-	 - transfile    - path/filename of transaction file
-	 - noresults    - no of results to save in results log file
-	 - const_k1     - value for K1 constant (BM25)
-	 - const_b      - value for B constant (BM25)
-	 - topicfile    - path/filename of topic file for which relevance judgement file is available
-	 - topicfields  - fields of topic to use from topic file in evaluation
-	 - relfile      - path/filename of relevance judgements file
-	 - runname      - name of the evaluation run
-	 - nterms       - no of terms to pick from the topic
-	 - stopsfile    -  name of the stopword file
-	 - evaluationfiles - path/filename of the evaluation files(where evaluation result is stored).
-	 - indexbigram  - Index Bigram in the index.
-	 - queryparsebigram -  Parse and add bigrams to the Query.
-	 - weightingscheme - which weighting scheme to select(bm25,lmweight,trad,boolweight)
-
-Parameters to be configured using for BM25 Weighting Scheme
+	 - textfile     - (INPUT) path/filename of text file to be indexed for evaluation
+	 - language     - (INPUT) corpus language to be indexed for stemming
+	 - db           - (OUTPUT) path of database which is index of above textfile
+	 - querytype    - (INPUT) type of query
+	 - queryfile    - (OUTPUT) path/filename of query file to be ran and searched
+	 - resultsfile  - (OUTPUT) path/filename of results file(run file)
+	 - transfile    - (INPUT) path/filename of transaction file
+	 - noresults    - (INPUT) no of results to save in results log file
+	 - const_k1     - (INPUT) value for K1 constant (BM25)
+	 - const_b      - (INPUT) value for B constant (BM25)
+	 - topicfile    - (INPUT) path/filename of topic file for which relevance judgement file is available
+	 - topicfields  - (INPUT) fields of topic to use from topic file in evaluation
+	 - relfile      - (INPUT) path/filename of relevance judgements file
+	 - runname      - (INPUT) name of the evaluation run
+	 - nterms       - (INPUT) no of terms to pick from the topic
+	 - stopsfile    - (INPUT) name of the stopword file
+	 - evaluationfiles - (OUTPUT) path/filename of the evaluation files(where evaluation result is stored).
+	 - indexbigram  - (INPUT) Index Bigram in the index.
+	 - queryparsebigram -(INPUT) Parse and add bigrams to the Query.
+	 - weightingscheme - (INPUT) which weighting scheme to select(bm25, lmweight, tradweight, boolweight)
```

Comment:
Looking at the code, this change is wrong - `"trad"` is what is checked for.

Reply:
Corrected.